app/views/facet_taggings/_tagging_form.html.erb

```diff
@@ -20,7 +20,6 @@
 
   <hr/>
 
-
```

Comment:
Was this line removed because of a lint error? Can it be fixed up?

Reply:
Yes and no I can't fix it up _easily_, not without unpicking the validation changes in the same file.